namara/__init__.py

```diff
@@ -10,7 +13,23 @@ def __init__(self, api_key, debug=False, host='https://api.namara.io', api_versi
         self.api_version = api_version
         self.base_path = '{0}/{1}'.format(self.host, self.api_version)
         self.headers = {'Content-Type': 'application/json', 'X-API-Key': api_key}
-
+    
+    def export_file_to_dataframe(self, dataset_id, organization_id, project_id, options=None):
+        list_of_chunks = [] 
+        url = self.get_url(f'/data_sets/{dataset_id}/data/export?geometry_format=wkt&organization_id={organization_id}&project_id={project_id}')
+        while True:
+            response = self.__session.get(url, params=options, headers=self.headers).result().json()
+            if self.debug: 
+                logging.debug('REQUEST URL: ' + response['url'])
+            if response['message'] == 'Exported':
+                for chunk in pd.read_csv(response['url'], chunksize=100):
```

Comment:
If `output_format == 'url'` return the URL
If `output_format == 'dataframe'` do this.